supabase/functions/check-congestion-charge/index.ts

```diff
@@ -0,0 +1,47 @@
+// @ts-nocheck (for Deno references, annoyingly cannot get around for now)
+/* global Deno */
+
+import { type Handler, serve } from "https://deno.land/std@0.168.0/http/server.ts";
+import { createClient } from "https://esm.sh/@supabase/supabase-js@latest";
+import { corsHeaders } from "../_shared/cors.ts";
+
+serve(async (req: Handler): Promise<Response> => {
+    if (req.method === "OPTIONS") {
+        return new Response("ok", { headers: corsHeaders });
+    }
+
+    const request = await req.json();
+
+    const generateHeaders = (status: number = 200): any => ({
+        ...corsHeaders,
+        "Content-Type": "application/json",
+        status,
+    });
+
+    const SUPABASE_URL = Deno.env.get("SUPABASE_URL");
+    const SUPABASE_ANON_KEY = Deno.env.get("SUPABASE_ANON_KEY");
+    const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY, {
+        global: {
+            headers: {
+                Authorization: req.headers.get("Authorization")!,
+            },
+        },
+    });
+
+    const { data, error } = await supabase.storage
+        .from("public")
+        .download("public/congestion-zone-postcodes.txt");
```

Comment:
Do you think it would be worth it to have this data stored within the database itself for performance in future? I don't think it needs to be done just now, I just wanted to know what you thought

Reply:
Actually, as long as a set of postcodes are processed in bulk, I think this should be fine